coverage/src/BamView-WebComponent.js

```diff
@@ -256,10 +288,15 @@ class BamViewChart extends HTMLElement {
                 alert(`Gene ${geneName} is not in ${source} for build ${build}`);
                 return;
             }
-            const chr = data[0].chr;
+
+            const rname = data[0].chr;
             const start = parseInt(data[0].start);
             const end = parseInt(data[0].end);
-            this._bamView.zoomToChromomsomeRegion(this.validBamReadDepth, chr, start, end, geneName);
+
+            this._rname = rname;
```

Comment:
These 3 lines seem to be duplicated a lot. Would it maybe make more sense to save the values in `zoomToChromosomeRegion`? That might only save one duplicate, but it's worth thinking about where the values are actually used and whether you can clean it up at all.